File: src/app/dtos/personnel/CreatePersonnelDto.ts
export class CreatePersonnelDto {
  identityNumber: string = "";
  firstName: string = "";
  lastName: string = "";
  email: string = "";
  phoneNumber: string = "";
  isBanned: boolean = false;
  personnelBranches?: string[]; // Guid -> string olarak tutulur
}

File: OskApi_temp/Dtos/Personnel/CreatePersonnelDto.cs
using OskApi.Entities;
using OskApi.Entities.Personnel;

namespace OskApi.Dtos.Personnel
{
    public class CreatePersonnelDto
    {
      
            public string IdentityNumber { get; set; } = string.Empty;
            public string FirstName { get; set; } = string.Empty;
            public string LastName { get; set; } = string.Empty;
            public string Email { get; set; } = string.Empty;
            public string PhoneNumber { get; set; } = string.Empty;
            public bool IsBanned { get; set; } = false;
            public ICollection<Guid>? PersonnelBranches { get; set; }


        
    }
}
